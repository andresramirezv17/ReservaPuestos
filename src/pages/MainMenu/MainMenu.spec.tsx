import React from 'react';
import { render, fireEvent } from '@testing-library/react';
import { MainMenu } from '.';

describe('MainMenu test', () => {
  it('should match with snapshot', () => {
    const { container } = render(<MainMenu />);
    expect(container).toMatchSnapshot();
  });
});
