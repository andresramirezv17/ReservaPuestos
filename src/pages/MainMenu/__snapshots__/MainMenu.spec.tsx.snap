// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainMenu test should match with snapshot 1`] = `
<div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthMd"
  >
    <div
      class="MuiTabs-root"
      style="border-radius: 10px;"
    >
      <div
        class="MuiTabs-scroller MuiTabs-fixed"
        style="overflow: hidden;"
      >
        <div
          class="MuiTabs-flexContainer MuiTabs-centered"
          role="tablist"
        >
          <button
            aria-selected="true"
            class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit Mui-selected MuiTab-labelIcon"
            role="tab"
            style="min-width: 40%; font-weight: bold; color: white;"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiTab-wrapper"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root makeStyles-iconLeft-1"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M4 18v3h3v-3h10v3h3v-6H4v3zm15-8h3v3h-3v-3zM2 10h3v3H2v-3zm15 3H7V5c0-1.1.9-2 2-2h6c1.1 0 2 .9 2 2v8z"
                />
              </svg>
              My Workspaces
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
          <button
            aria-selected="false"
            class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit MuiTab-labelIcon"
            role="tab"
            style="min-width: 40%; font-weight: bold;"
            tabindex="-1"
            type="button"
          >
            <span
              class="MuiTab-wrapper"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root makeStyles-iconRight-2"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 7V3H2v18h20V7H12zM6 19H4v-2h2v2zm0-4H4v-2h2v2zm0-4H4V9h2v2zm0-4H4V5h2v2zm4 12H8v-2h2v2zm0-4H8v-2h2v2zm0-4H8V9h2v2zm0-4H8V5h2v2zm10 12h-8v-2h2v-2h-2v-2h2v-2h-2V9h8v10zm-2-8h-2v2h2v-2zm0 4h-2v2h2v-2z"
                />
              </svg>
              Reserve
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </div>
        <span
          class="PrivateTabIndicator-root-4 PrivateTabIndicator-colorSecondary-6 MuiTabs-indicator"
          style="left: 0px; width: 0px;"
        />
      </div>
    </div>
    <div>
      <h5
        class="MuiTypography-root makeStyles-title-8 MuiTypography-h5"
      >
        Puestos reservados
      </h5>
      <p
        class="MuiTypography-root MuiTypography-body1"
      >
        Estos son los espacios que has reservado
      </p>
      <div
        style="width: 100%; display: inline-flex; flex-wrap: wrap; gap: 15px;"
      >
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3"
        />
      </div>
      <div
        class="makeStyles-boxNoReserves-9"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root"
          focusable="false"
          style="font-size: 5em; color: rgb(9, 38, 121);"
          viewBox="0 0 24 24"
        >
          <path
            d="M23 16c0 1.1-.9 2-2 2h-1l-2-2h3V4H6L4 2h17c1.1 0 2 .9 2 2v12zm-5.5 2l-2-2zm-2.6 0l6 6 1.3-1.3-4.7-4.7-2-2L1.2 1.8 0 3.1l1 1V16c0 1.1.9 2 2 2h7v2H8v2h8v-2h-2v-2h.9zM3 16V6.1l9.9 9.9H3z"
          />
        </svg>
        <p
          class="MuiTypography-root makeStyles-noReservesText-10 MuiTypography-body1"
        >
          Todavia no tienes reservas
        </p>
      </div>
    </div>
  </div>
</div>
`;
